src/unwinder/find_fde/custom.rs

```diff
@@ -102,19 +102,18 @@ fn find_fde<T: EhFrameFinder + ?Sized>(eh_frame_finder: &T, pc: usize) -> Option
 
 fn find_fde_with_eh_frame_hdr(
     pc: usize,
-    text_base: usize,
+    text_base: Option<usize>,
     eh_frame_hdr: usize,
 ) -> Option<FDESearchResult> {
     unsafe {
-        let bases = BaseAddresses::default()
-            .set_text(text_base as _)
-            .set_eh_frame_hdr(eh_frame_hdr as _);
-        let eh_frame_hdr = EhFrameHdr::new(
-            get_unlimited_slice(eh_frame_hdr as usize as _),
-            NativeEndian,
-        )
-        .parse(&bases, core::mem::size_of::<usize>() as _)
-        .ok()?;
+        let mut bases = BaseAddresses::default().set_eh_frame_hdr(eh_frame_hdr as _);
+        if let Some(text_base) = text_base {
+            bases = bases.set_text(text_base as _);
+        }
+        let eh_frame_hdr =
```

Comment:
On my machine this change is undone by running `cargo fmt`. Did you have a global rustfmt config?

Reply:
I have it as a subdirectory of the `rust` project, and I think they use some nonstandard settings that `rustfmt` must have picked up even though it's a different repository.

I've moved it out of the `rust` directory and re-run `cargo fmt` without anything in any directories above it.